mobile/src/main/java/com/ataulm/talkbacktile/Toggler.java

```diff
@@ -51,6 +51,11 @@ void disableTalkBack() {
         Settings.Secure.putString(contentResolver, Settings.Secure.ACCESSIBILITY_ENABLED, value);
     }
 
+    private String getEnabledAccessibilityServices() {
+        String value = Settings.Secure.getString(contentResolver, Settings.Secure.ENABLED_ACCESSIBILITY_SERVICES);
+        return value == null ? "" : value;
```

Comment:
noice, I guess mine were never null because I was experimenting on the device before this 😬